fundingmanager.go

```diff
@@ -1212,18 +1221,33 @@ func (f *fundingManager) handleFundingOpen(fmsg *fundingOpenMsg) {
 	)
 	tweaklessCommitment := localTweakless && remoteTweakless
 	chainHash := chainhash.Hash(msg.ChainHash)
+
+	// If the peer sent an upfront shutdown script, but we do not support the
+	// feature, log a warning. Enforcement of this script occurs entirely on
+	// our side, so we can ignore it.
+	localUpfrontShutdown := fmsg.peer.LocalLocalFeatures().HasFeature(
```

Comment:
is this configurable now? If its not I guess we don't have to do this check. 

It should probably be configurable though :)

Reply:
It should be, I'll add a `enable_upfront_script` option?
I was thinking that it would be cool if people could specify an upfront shutdown addr when they call `openChannel` from lncli/rpc but that requires knowledge of peer feature bits etc so I think I'll leave it simple for now.